src/User/UserServiceProvider.php

```diff
@@ -82,6 +96,16 @@ public function boot()
             return false;
         });
 
+        $passwordCheckers = array_map(function ($checker) {
+            if (is_string($checker)) {
+                $checker = $this->app->make($checker);
+            }
+
+            return $checker;
+        }, $this->app->make('flarum.user.password_checkers'));
```

Comment:
I've seen similar logic before, can't we make a helper function that loops and resolves automatically?

Reply:
We could, but considering the need for imports, I don't know how much better it would be. Also unsure where we would put it. Regardless, I'd rather revisit this after we finish extenders.